torchci/lib/fetchPR.ts

```diff
@@ -51,7 +51,7 @@ export default async function fetchPR(
   repo: string,
   prNumber: string,
   octokit: Octokit,
-  useClickhouse: boolean = false
+  useClickhouse: boolean = true
```

Comment:
I'm pretty sure this will break anything that hasn't been migrated yet, but if it already errors I guess it doesn't matter